src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -88,19 +88,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     String user = userService.getCurrentUser().getEmail();
     
- // Get the message entered by the user.
+  // Get the message entered by the user.
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
-    
     String regex = "(https?://\\S+\\.(png|jpg))";
-    String replacement = "<img src=\"$1\" />";
-    String textWithImagesReplaced = text.replaceAll(regex, replacement);
 
-    // Get the URL of the image that the user uploaded to Blobstore.
-    String imageUrl = getUploadedFileUrl(request, "image");
```

Comment:
I had to tweak this a little bit because I was getting an error when the message doesnt contain any images(Blobstore couldnt find any images so it would throw an error). This code checks if theres an image tag in the text before attempting to upload to blobstore.